plugin/src/main/scala/org/ehcache/integrations/play/EhcacheJCacheWrapperModule.scala

```diff
@@ -63,11 +83,34 @@ class EhcacheValueWrapper extends ValueWrapper {
   */
 class EhcacheJCacheWrapper(xmlConfig: Option[XmlConfiguration]) extends JCacheWrapper {
 
+  var enhanced = Set.empty[String]
+
   def valueWrapper(name: String) = {
-    new EhcacheValueWrapper
+    enhanced contains name match {
+      case true => new EhcacheValueWrapper
+      case false => new NoOpValueWrapper
+    }
   }
 
   def enhanceConfiguration(name: String, baseConfig: JCacheConfiguration[String, Any]): JCacheConfiguration[String, Any] = {
-    baseConfig
+    xmlConfig match {
+      case Some(config) =>
+        val template: CacheConfigurationBuilder[String, Any] = config
```

Comment:
Don't see the point of wrapping just to unwrap behind. This does not leave the scope of the method.